runtime/src/lib.rs

```diff
@@ -464,6 +467,7 @@ impl pallet_collator_selection::Config for Runtime {
 impl pallet_sudo::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type RuntimeCall = RuntimeCall;
+	type WeightInfo = ();
```

Comment:
isn't this a DDOS enabler?

Reply:
Yeah .. I created new issue for this https://github.com/Polkadex-Substrate/parachain/issues/64